metadata.json

```diff
@@ -1,25 +1,22 @@
 {
-    "KPlugin": {
-        "Category": "Appearance",
-        "Description": "Animation for windows moved or resized by programs or scripts",
-        "EnabledByDefault": false,
-        "Id": "kwin4_effect_geometry_change",
-        "License": "GPL",
-        "Name": "Geometry Change",
-        "ServiceTypes": [
-            "KWin/Effect"
-        ],
-        "Version": "1",
-        "Website": "",
-        "Authors": [
-            {
-                "Email": "peter.fajdiga@gmail.com",
-                "Name": "Peter Fajdiga"
-            }
-        ],
-    },
-    "X-KDE-Ordering": "60",
```

Comment:
Do you perhaps know what would be an appropriate value here?

Reply:
The ordering is relative to other scripts trying to animate the same thing, but since there does not seem to be any other script that do this in the store, the value becomes irrelevant. You can add back the property with any value just so other scripts can order themselves relative to you. Search for "X-KDE-Ordering" here to see the guy explaining a fix to a problem: https://store.kde.org/p/1081235. In short, this number is rather arbitrary and only become sensible if another script appears which animate the same thing.